SQL/SQL (basic).sublime-syntax

```diff
@@ -290,7 +294,7 @@ contexts:
     - include: immediately-pop
 
   drop-other:
-    - match: \b(?i:user)\b(?!\s+(?i:if)\b)
```

Comment:
having the `\b` outside the lookahead gives it a different meaning, is that intentional? it will not match usernames beginning with `if`  like `drop user iffy` instead of i.e. `drop user if exists iffy`

Reply:
According to https://github.com/forkeith/Packages/pull/28/commits/209885543d4ea649db3553f8fad300e56e9752c2 it doesn't seem to make a difference.